src/app/pages/settings.cpp

```diff
@@ -274,6 +276,12 @@ QWidget *LayoutSettingsTab::settings_widget()
     layout->addWidget(Session::Forge::br(), 1);
     layout->addWidget(this->control_bar_widget(), 1);
 
+    layout->addWidget(Session::Forge::br(), 1);
```

Comment:
youll want to add these after `controls_bar_row` ('Quick View' should come right after 'Control Bar')

![image](https://user-images.githubusercontent.com/16892670/149075350-7aa5df95-f25e-4c7d-b352-4c9342e40802.png)


and you should do something similar to `controls_bar_row` where you only show it if full screen mode is on